dsn.go

```diff
@@ -560,7 +560,7 @@ func parseDSNParams(cfg *Config, params string) (err error) {
 				} else {
 					cfg.TLSConfig = "false"
 				}
-			} else if vl := strings.ToLower(value); vl == "skip-verify" {
+			} else if vl := strings.ToLower(value); vl == "skip-verify" || vl == "optional" {
```

Comment:
Should this really set `cfg.tls = &tls.Config{InsecureSkipVerify: true}` ?

Reply:
It doesn't have to. but then there should be an option to set `tls=optional` and also supply a config for the CA etc.

Reply:
This option is not for security.  If security is matter, they shouldn't allow unencrypted connection.  In unencrypted connection, there are no verify for host.

This option is very silly behavior.  It useful only for situations described in #899.
So `InsecureSkipVerify: true` make sense.

Reply:
Please add a note to the README that using this option does not add any security.
Indeed, a Man-in-the-Middle attacker could remove TLS entirely with this option, whether `InsecureSkipVerify: true` is used or not.